Kickstarter-iOS/Views/Controllers/ManageViewPledgeViewController.swift

```diff
@@ -3,15 +3,15 @@ import Library
 import Prelude
 import UIKit
 
-final class ManagePledgeViewController: UIViewController {
+final class ManageViewPledgeViewController: UIViewController {
```

Comment:
I think the old name is better 😬 what made you change it to `ManageViewPledgeViewController`?

Reply:
The main reason is that we will use the same screen to Manage and View pledge.

Reply:
Okay makes sense! 👍